packages/components/Callout/macos/Callout.swift

```diff
@@ -289,15 +289,37 @@ class CalloutView: RCTView, CalloutWindowLifeCycleDelegate {
 	private var proxyView: RCTView = RCTView()
 
 	private lazy var calloutWindow: CalloutWindow = {
-		let calloutWindowController = CalloutWindowRootViewController()
-		let window = CalloutWindow(contentViewController: calloutWindowController)
+		let window = CalloutWindow()
 		window.windowLifeCycleDelegate = self
 		window.isReleasedWhenClosed = true
+
 		window.styleMask = [.fullSizeContentView]
 		window.level = .popUpMenu
+		window.backgroundColor = .clear
+		window.isMovableByWindowBackground = false
+		
+		let visualEffect = NSVisualEffectView()
+		visualEffect.translatesAutoresizingMaskIntoConstraints = false
+		visualEffect.material = .menu
+		visualEffect.state = .active
+		visualEffect.wantsLayer = true
+		visualEffect.layer?.cornerRadius = 5.0
+
+
```

Comment:
nit- did you mean to have a double newline inside this method?

Reply:
No, I'll fix that